src/main/modules/i18next/index.ts

```diff
@@ -0,0 +1,27 @@
+const i18next = require('i18next');
+const i18nextMiddleware = require('i18next-http-middleware');
+import express = require('express');
+
+const requireDir = require('require-directory');
+const resources = requireDir(module, '../../', {include: /locales/}).locales;
+const merge = require('lodash').merge;
+
+export class I18next {
+
+  constructor() {
+    const options = {preload: ['en', 'cy'], resources, debug: true, fallbackLng: 'en', supportedLngs:['en','cy'], detection: {order: ['querystring']}};
+
+    i18next
+      .use(i18nextMiddleware.LanguageDetector)
+      .init(options);
+  }
+
+  public enableFor(app: express.Express): void {
+    app.use(i18nextMiddleware.handle(i18next));
+    app.use((req:any, res, next) => {
+      const data = req.i18n.getDataByLanguage(req.lng).template;
+      merge(res.locals, data);
```

Comment:
Maybe we should use `Object.assign({}, res.locals, data)` rather than lodash?

Reply:
Done.